src/current/v23.1/node-shutdown.md

```diff
@@ -76,31 +89,35 @@ Node drain consists of the following consecutive phases:
 
     <section class="filter-content" markdown="1" data-scope="decommission">
-    Since all range replicas were already removed from the node during the [decommissioning](#decommissioning) stage, this step immediately resolves.
+    Since all range replicas were already removed from the node during the [draining](#draining) stage, this step immediately resolves.
     </section>
 
 <section class="filter-content" markdown="1" data-scope="drain">
-When [draining manually](#drain-a-node-manually), if the above steps have not completed after 10 minutes by default, node draining will stop and must be restarted to continue. For more information, see [Drain timeout](#drain-timeout).
+When [draining manually](#drain-a-node-manually), if the above steps have not completed after [`server.shutdown.drain_wait`](#server-shutdown-drain_wait) (10 minutes by default), node draining will stop and must be restarted manually to continue. For more information, see [Drain timeout](#drain-timeout).
 </section>
 
 <section class="filter-content" markdown="1" data-scope="decommission">
 #### Status change
 
-After decommissioning and draining are both complete, the node membership changes from `decommissioning` to `decommissioned`.
+After the node is drained and decommissioned, the node membership status changes from `decommissioning` to `decommissioned`.
 
-This node is now cut off from communicating with the rest of the cluster. A client attempting to connect to a `decommissioned` node and run a query will get an error.
+This node is now prevented from communicating with the rest of the cluster, and client connection attempts to the node will result in an error.
 </section>
 
-At this point, the `cockroach` process is still running. It is only stopped by process termination.
+At this point, it is safe to terminate the `cockroach` process manually or using your process manager or other deployment tooling (such as Kubernetes, Nomad, or Docker).
 
 #### Process termination
 
 <section class="filter-content" markdown="1" data-scope="decommission">
-An operator [terminates the node process](#terminate-the-node-process).
+After draining and decommissioning are complete, an operator [terminates the node process](#terminate-the-node-process).
 </section>
 
 <section class="filter-content" markdown="1" data-scope="drain">
-After draining completes, the node process is automatically terminated (unless the [node was manually drained](#drain-a-node-manually)).
+After draining is complete:
+
+- If the node was drained automatically because the `cockroach` process received a `SIGTERM` signal, the `cockroach` process is automatically terminated when draining is complete.
+- If the node was drained manually because an operator issued a `cockroach node drain` command, the `cockroach` process must be terminated manually. A minimum of 60 seconds after draining is complete, send it a `SIGTERM` signal to terminate it. Refer to [Terminate the node process](#terminate-the-node-process).
```

Comment:
```suggestion
- If the node was drained manually because an operator issued a `cockroach node drain` command, the `cockroach` process must be terminated manually. A minimum of 60 seconds after draining is complete, send it a `SIGTERM` signal to terminate it. Refer to [Terminate the node process](#drain-the-node-and-terminate-the-node-process).
```